.github/workflows/license_scan.yml

```diff
@@ -0,0 +1,33 @@
+name: Trivy License Scans
+
+on:
+  pull_request:
+    branches: [ "main" ]
+
+
+jobs:
+  build:
```

Comment:
Is this truly a build task? Let's call it what it is license_scan vs security_scan or trivy_scans if it's both.